### PR TITLE
Text data storing and retrieving from/to browser

### PR DESCRIPTION
**Pull request by sohanemon**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,45 +1,52 @@
-import React, { useRef } from 'react';
-import Editor, { Monaco } from '@monaco-editor/react';
+import { Loader } from '@/components/loader';
```
Comment (salesforce-casts) [inline pages/index.js:1]: the deployment failed, should the package.json include loader dependency as well?  @sohanemon 

Screenshot 2023-08-29 at 14 02 15

Comment (sohanemon) [reply pages/index.js:1]: loader isn't any dependency there. I have created a new component named loader.

Comment (salesforce-casts) [reply pages/index.js:1]: I don't see any file with the name loader.js and neither do i find the folder components in the branch. am i missing something?

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+components/loader.js
```
Comment (salesforce-casts) [inline .gitignore]: is it because of this am not finding it in the PR? @sohanemon

**Pull request merged**